src/Razor/test/RazorLanguage.Test/IntegrationTests/ComponentTemplateIntegrationTest.cs

```diff
@@ -58,8 +58,10 @@ public class MyComponent : ComponentBase
             var generated = CompileToCSharp(@"<MyComponent attr=""@<div></div>"" />");
 
             // Assert
-            var diagnostic = Assert.Single(generated.Diagnostics);
-            Assert.Equal("RZ1005", diagnostic.Id);
+            Assert.Collection(
+                generated.Diagnostics,
+                d => Assert.Equal("RZ10014", d.Id),
```

Comment:
hmmm whatsit?

Reply:
Ah yes. This was because `UseTwoPhaseCompilation` was set to `false`. I'll change it.